modules/map/tools/map_datachecker/server/alignment.hpp

```diff
@@ -0,0 +1,152 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#ifndef _MODULES_MAP_TOOLS_MAP_DATACHECKER_ALIGNMENT_HPP
+#define _MODULES_MAP_TOOLS_MAP_DATACHECKER_ALIGNMENT_HPP
+#include <memory>
+#include <vector>
+#include "modules/map/tools/map_datachecker/server/common.hpp"
+#include "modules/map/tools/map_datachecker/proto/collection_error_code.pb.h"
+
+namespace apollo {
+namespace hdmap {
+
+typedef struct BadOrGoodPoseInfo {
+  BadOrGoodPoseInfo(): start_time(-1.0), end_time(-1.0), pose_count(0) {}
+  double start_time, end_time;
+  int pose_count;
+} BadOrGoodPoseInfo;
+
+class Alignment {
+ public:
+  explicit Alignment(std::shared_ptr<JSonConf> sp_conf) {
```

Comment:
Use constructor member initializer lists instead of assigning value, like what you dit at Line 27 for BadOrGoodPoseInfo.